modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/RowHandler.java

```diff
@@ -17,36 +17,75 @@
 
 package org.apache.ignite.internal.sql.engine.exec;
 
+import java.nio.ByteBuffer;
 import org.apache.ignite.internal.sql.engine.exec.row.RowSchema;
 import org.jetbrains.annotations.Nullable;
 
 /**
  * Universal accessor and mutator for rows. It also has factory methods.
  */
 public interface RowHandler<RowT> {
+    /**
+     * Extract appropriate field.
+     *
+     * @param field position.
```

Comment:
```suggestion
     * @param field Field position.
```
Do you mean field index or offset in byte array, if RowT is byte array?

Reply:
RowT is not instantiated by ByteArray for now, we have no such implementation.
Java doc changed.